code/lib/builder-webpack5/src/preview/babel-loader-preview.ts

```diff
@@ -10,6 +10,7 @@ export const createBabelLoader = (options: any, typescriptOptions: TypescriptOpt
         options,
       },
     ],
+    resourceQuery: { not: [/raw/] },
```

Comment:
Hey @IanVS is there a Vite counterpart for this setting (or is this just default?)

Reply:
I'm not sure, what does this webpack setting do?  If you add `?raw` to the end of an import in vite,  it will not be processed by vite and the source itself will be imported, if that's related at all...

Reply:
Yup I think that's the intended behavior! So it's just out of the box with Vite then